.gitignore

```diff
@@ -0,0 +1,33 @@
+# Default .gitignore for Ruby
+*.gem
+*.rbc
+.bundle
+.config
+coverage
+InstalledFiles
+lib/bundler/man
+pkg
+rdoc
+spec/reports
+test/tmp
+test/version_tmp
+tmp
+Gemfile.lock
+
+# YARD artifacts
+.yardoc
+_yardoc
+doc/
+
+# Vim
+*.swp
+
+# OS X
+.DS_Store
+
+# Puppet
+coverage/
+spec/fixtures/modules/*
+spec/fixtures/manifests/*
+# needed to spec test alternative templates
+!spec/fixtures/modules/my_rngd
```

Comment:
my_rngd contains an alternative template. It is needed to test $config_template at https://github.com/Phil-Friderici/puppet-rngd/blob/master/spec/classes/init_spec.rb#L58-L74